client/src/components/Chat.jsx

```diff
@@ -329,12 +330,21 @@ const Chat = () => {
 
     };
 
+    const adjustTextareaHeight = () => {
+        if (inputRef.current) {
+          inputRef.current.style.height = '45px';
```

Comment:
use a relative value instead

Reply:
Yeah I really had no idea where that file came from tried deleting it

Reply:
@Dun-sin you can check it again now, just committed
![Chat jsx - Whisper - Visual Studio Code 10_9_2023 10_33_39 AM](https://github.com/Dun-sin/Whisper/assets/122406759/f1e173d1-9121-4493-8ebf-84189440a1cc)